pkg/keyspace/keyspace.go

```diff
@@ -68,6 +72,7 @@ type Manager struct {
 	ctx context.Context
 	// config is the configurations of the manager.
 	config Config
+	kgm    *GroupManager
```

Comment:
This manager has the same name as:

https://github.com/tikv/pd/blob/9fcde123fd7c7baddc6d1898dfc07cf2a461f6f7/pkg/tso/keyspace_group_manager.go#L57

I strongly feel like we should unify them or give this keyspace group storage manager another name.

Reply:
They are two different things. And any suggestion for the name?